model/src/application/interaction/message_component.rs

```diff
@@ -1,16 +1,34 @@
+//! Data received when an [`MessageComponent`] interaction is executed.
+//!
+//! See [Discord Docs/Message Component Data Structure].
+//!
+//! [`MessageComponent`]: crate::application::interaction::InteractionType::MessageComponent
+//! [Discord Docs/Message Component Data Structure]: https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-object-message-component-data-structure
```

Comment:
Same as `ApplicationCommand`
```suggestion
//! [`MessageComponent`] interaction.
//!
//! [`MessageComponent`]: crate::application::interaction::InteractionType::MessageComponent
```